include/evmc/evmc.hpp

```diff
@@ -159,10 +159,9 @@ inline constexpr uint64_t fnv1a_by64(uint64_t h, uint64_t x) noexcept
 /// The "equal to" comparison operator for the evmc::address type.
 inline constexpr bool operator==(const address& a, const address& b) noexcept
 {
-    // TODO: Report bug in clang keeping unnecessary bswap.
-    return load64be(&a.bytes[0]) == load64be(&b.bytes[0]) &&
-           load64be(&a.bytes[8]) == load64be(&b.bytes[8]) &&
-           load32be(&a.bytes[16]) == load32be(&b.bytes[16]);
+    return load64le(&a.bytes[0]) == load64le(&b.bytes[0]) &&
```

Comment:
Why is this faster? Is the `load32le` helper recognized to be a simple `mov`/`lea`?

Reply:
This omits `bswap`.

Reply:
That's what I asked "recognized to be a simple mov".

Reply:
Yes. The `load64le` is rather `mov` instead of `mov` + `bswap` or `movbe`.

Reply:
However, GCC can recognize that `bswap(a) == bswap(b)` is actually `a == b`. So this only helps Clang.